### PR TITLE
Scroll to next/previous thread when pressing j/k (fixes #3327)

### LINKED ISSUE #3327: Keyboard shortcut to  skip to the next post
Hello,

Because post can be pretty long (and have a lot of comments), I find myself doing a lot of scrolling when I browser my network tab.
I think it would be useful to have a keyboard shortcut that scrolls the next post into view.
What do you think?

### PR DESCRIPTION
**Pull request by Rudloff**
This patch adds keyboard events on the `j` and `k` keys that scroll to the next/previous thread.
I have successfully tested it with the vier and frio themes.

### CODE REVIEW

**Comment by Hypolite** [comment]
Confirmed, this is awesome, can you please add this behavior in `doc/Accesskeys.md`?

**Comment by Rudloff** [comment]
I updated the PR to ignore the event when it comes from a `textarea` or `input` element.

**Comment by Rudloff** [comment]
@Hypolite Accesskeys are a different mechanism so I think it might be confusing to document this in the same list.

File: js/main.js [hunk 1]
```diff
@@ -322,6 +322,29 @@
 			}
 		});
 
+		$(document).keydown(function (event) {
```
Comment (rabuzarus) [inline js/main.js]: can you please add a little comment on top which does describe what this code part is about

**Comment by Hypolite** [comment]
> @Hypolite Accesskeys are a different mechanism so I think it might be confusing to document this in the same list.

This is true, can you create a new documentation file `doc/KeyboardShortcuts.md` based on `doc/Accesskeys.md`?

**Comment by Hypolite** [comment]
Also in the future can you avoid amending your previous commits? It helps us review the exact changes you did between reviews. For example, I had to go over the whole diff to find the one line you changed to prevent scrolling in `input` and `textarea`.

**Comment by AlfredSK** [comment]
A dream came true. Thanks a lot! :+1:

**Pull request merged**